src/server_manager/web_app/display_server.ts

```diff
@@ -13,13 +13,7 @@
 // limitations under the License.
 
 import * as server from '../model/server';
-
-export interface DisplayServer {
```

Comment:
Why are you moving the definition to app root?

Reply:
This is really a concept owned by the AppRoot. It's part of its interface to the outside world. Furthermore, I thought Javascript couldn't depend on Typescript, but it can!

Since we have a display_server file, I've moved it back to where it was.